src/controllers/update/confirm.overseas.entity.details.controller.ts

```diff
@@ -15,13 +16,25 @@ export const get = (req: Request, res: Response, next: NextFunction) => {
     const appData: ApplicationData = getApplicationData(req.session as Session);
     const update = appData.update as Update;
 
+    if (isRemoveJourney(req)) {
+      return res.render(config.CONFIRM_OVERSEAS_ENTITY_DETAILS_PAGE, {
+        journey: config.JourneyType.remove,
+        backLinkUrl: `${config.OVERSEAS_ENTITY_QUERY_URL}${config.JOURNEY_REMOVE_QUERY_PARAM}`,
+        updateUrl: config.UPDATE_OVERSEAS_ENTITY_CONFIRM_URL,
+        templateName: config.CONFIRM_OVERSEAS_ENTITY_DETAILS_PAGE,
+        appData,
+        registrationDate: update.date_of_creation,
+      });
+    }
+
     return res.render(config.CONFIRM_OVERSEAS_ENTITY_DETAILS_PAGE, {
       backLinkUrl: config.OVERSEAS_ENTITY_QUERY_URL,
       updateUrl: config.UPDATE_OVERSEAS_ENTITY_CONFIRM_URL,
       templateName: config.CONFIRM_OVERSEAS_ENTITY_DETAILS_PAGE,
       appData,
-      registrationDate: update.date_of_creation
+      registrationDate: update.date_of_creation,
```

Comment:
i'd remove the comma that was added here so it is the same as before and won't show up as a change